projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-data.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { TzDatepickerModule } from './tz-datepicker.module';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: TzDatepickerModule
+})
+export class TzDataService {
+
+  constructor(private http: HttpClient) {
+    this.http = http
+  }
+
+  fetchTzs() {
+    return this.http.get<{ version: string; links: string[]; zones: string[] }>('/assets/data/timezones.json')
```

Comment:
Is there a reason this needs to be loaded over http? It gets deployed with the application, so why not just hard code it? It seems like another asset we'll have to host in downstream applications.

Reply:
The main reason I did this was because of size. The Moment timezone data is pushing a megabyte (that's more than 1/3rd the size of Publish's minified codebase, for example). This approach allows loading that content and then displaying this specific control asynchronously, without blocking interaction with the rest of the page.

It is another asset that we have to host, but we already require that consumers host image assets from Styleguide, so it didn't seem like a big ask to me to add one more.